apps/api/src/bid/routes/bids/bids.router.ts

```diff
@@ -0,0 +1,34 @@
+import { createRoute } from "@hono/zod-openapi";
+import { container } from "tsyringe";
+
+import { BidController } from "@src/bid/controllers/bid/bid.controller";
+import { ListBidsParamsSchema, ListBidsResponseSchema } from "@src/bid/http-schemas/bid.schema";
+import { OpenApiHonoHandler } from "@src/core/services/open-api-hono-handler/open-api-hono-handler";
+
+const listRoute = createRoute({
+  method: "get",
+  path: "/v1/bids/{dseq}",
```

Comment:
`dseq` is not a unique identifier for a deployment. `userId` + `dseq` or `address/owner` + `dseq` is. In our case user seems to be more appropriate. So the interface should prob be something like:

```suggestion
  path: "/v1/bids/${userId}/{dseq}",
```

It's not perfect so it's totally open to discussion, however user should be taken into account

Reply:
Right, but if you [check the controller](https://github.com/jzsfkzm/console/blob/d4aa97f2d4b367a5c9340dd6e257f3292f3cd6b0/apps/api/src/bid/controllers/bid/bid.controller.ts#L18), we're grabbing the user's info there, leaning on x-api-key.
Isn't that what we need here?

Reply:
yes, I understand this. However it's generally better to specify a user id anyway and then use permissions to limit the query or check if authorized. this endpoint then could be used according to permissions.

Reply:
Wouldn't it be better to have dseq and userId as query string params?
`GET /v1/bids?dseq={dseq}&userId={userId}`

and in the menwhile
- dseq would be mandatory
- userId would be optional
  - when not set, we would load bids of the user in auth
  - when set, would be checked against auth

This way regular users would not need to send their userId, but we could still use it for admin purposes.

Reply:
yea, sounds good

Reply:
Made changes according to this, could you re-review pls?